docs/Changelog.md

```diff
@@ -28978,43 +28978,28 @@ This version of the operator has been available since version 23 of the default
 
 ### <a name="RMSNormalization-23"></a>**RMSNormalization-23**</a>
 
-  This is RMS normalization defined in ONNX as function.
-        The overall computation can be split into two stages.
-        The first stage is standardization, which makes the
-        normalized elements have zero mean and unit variances.
-        The computation required by standardization can be
+  This is RMS normalization defined in ONNX as function as described in the paper https://arxiv.org/pdf/1910.07467.
+        The overall computation can be split into two stages. The first stage is standardization, which makes the
+        normalized elements have zero mean and unit variances. The root mean squared norm is taken over the last D dimensions,
+        where D is the dimension of normalized_shape. For example, if normalized_shape is (3, 5) (a 2-dimensional shape),
+        the rms norm is computed over the last 2 dimensions of the input. The computation required by standardization can be
         described by the following equations.
         ```
-        Mean = ReduceMean<axes=normalized_axes>(X)
-        D = Sub(X, Mean)
-        DD = Mul(D, D)
-        Var = ReduceMean<axes=normalized_axes>(DD)
-        VarEps = Add(Var, epsilon)
-        StdDev = Sqrt(VarEps)
-        InvStdDev = Reciprocal(StdDev)
-        Normalized = Mul(D, InvStdDev)
+        XSquared = Mul(X, X)
```

Comment:
Can values be named with lower case, except for the inputs and outputs which should follow the convention https://onnx.ai/onnx/repo-docs/OpConventions.html

This way the values are not confused with the ops